ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.12.yaml

```diff
@@ -34,50 +34,67 @@ base_images:
 releases:
   intermediate413:
     candidate:
-      architecture: amd64
+      architecture: multi
       product: ocp
       stream: nightly
       version: "4.13"
   intermediate414:
     candidate:
-      architecture: amd64
+      architecture: multi
       product: ocp
       stream: nightly
       version: "4.14"
   intermediate415:
     candidate:
-      architecture: amd64
+      architecture: multi
       product: ocp
       stream: nightly
       version: "4.15"
+  intermediate416:
+    candidate:
+      architecture: multi
+      product: ocp
+      stream: nightly
+      version: "4.16"
   latest:
     release:
-      architecture: amd64
+      architecture: multi
```

Comment:
multiarch is GA starting from 4.13. So please drop the multiarch jobs in 4.12.

Reply:
updated the test matrix